src/main/java/calculator/Positive.java

```diff
@@ -0,0 +1,16 @@
+package calculator;
+
+public class Positive {
+    private int number;
```

Comment:
```suggestion
    private final int number;
```
불변 형태로 사용해서 부수효과를 줄여 보는 것은 어떨까요 ?! 

참고) https://velog.io/@gkskaks1004/%EB%B6%88%EB%B3%80-%EA%B0%9D%EC%B2%B4%EB%A5%BC-%EC%82%AC%EC%9A%A9%ED%95%B4%EC%95%BC%ED%95%98%EB%8A%94-%EC%9D%B4%EC%9C%A0%EC%99%80-%EB%B6%88%EB%B3%80-%EA%B0%9D%EC%B2%B4-%EC%83%9D%EC%84%B1-%EB%B0%A9%EB%B2%95